app/schemas.py

```diff
@@ -0,0 +1,9 @@
+from ninja import Schema
+
+
+class TranslationsSchema(Schema):
```

Comment:
Maybe a better name for this could be something that indicates a Translated String? Since this is basically a string multiplied by N languages

Reply:
:thinking: You are right to say that "translation" is a simplification, after all a "translation" could not only contain the translated text itself but also notes, a date, the translator's name etc. I find that in the given context, it is an acceptable simplification to assume that
- translated string
- == translated text
- == translation

I don't like using "string" in the name here, too implementation-specific. If you have another good suggestion let me know, otherwise I would propose to stick with "translations" for brevity.

Reply:
OK for me. I was rather thinking in the direction of it could be a TranslatedList or TranslatedAnything but maybe that's too far fetched.